misk-hibernate/src/main/kotlin/misk/jdbc/PingDatabaseService.kt

```diff
@@ -44,6 +44,7 @@ class PingDatabaseService @Inject constructor(
   }
 
   private fun connectToDataSource(dataSource: DriverDataSource) {
+    dataSource.setLoginTimeout((config.connection_timeout.toMillis() / 1000).toInt())
```

Comment:
If we move Misk to Java 11 (we should if we haven't already) this can be toSeconds().

Reply:
I know I tried that first. It failed in the javadocs.